src/wp-includes/class-wp-locale-switcher.php

```diff
@@ -186,7 +186,7 @@ public function is_switched() {
 	 *
 	 * @return string|false Locale if the locale has been switched, false otherwise.
 	 */
-	public function get_current_locale() {
+	public function get_locale() {
```

Comment:
Some more descriptive alternatives: 

```suggestion
	public function get_current_switched_to_locale() {
```

```suggestion
	public function get_switched_to_locale() {
```

```suggestion
	public function get_switched_locale() {
```

Reply:
Hmm if anything I'd use `get_switched_locale` / `get_switched_user_id`, but not sure if that's better over `get_locale` / `get_user_id`. After all these are methods on the `WP_Locale_Switcher` class, so having `_switched_` in the method name seems a bit redundant. Thoughts?

Reply:
Having a more descriptive method name would allow us to introduce more locale related methods, e.g. `get_previous_locale()`.

Reply:
Good point